### PR TITLE
Added an advanced method of making presets.md

### PR DESCRIPTION
**Pull request by Expertium**
@brishtibheja

### CODE REVIEW

**Comment by brishtibheja** [comment]
Change point three. Rewrite the first part as "Paste parameters-1 in the `FSRS parameters` field of preset-2". The sentence is a bit of a mess. One more thing, in the same third point clarify the purpose of doing `Evaluate`.

File: docs/tutorial.md [hunk 1]
```diff
@@ -108,7 +108,18 @@ If you have at least 1000/400 reviews (across all cards that this preset applies
...
+2) Create a new preset for the subdeck. Let's call it Preset-2.
+3) Paste Parameters-1 into the field with FSRS parameters of Preset-2 and click "Evaluate", write down the RMSE and log loss values.
+4) Click "Optimize" to obtain a new set of parameters for Preset-2. Let's call them Parameters-2.
+5) Click "Evaluate" and write down RMSE and log loss values.
+6) If Parameters-2 result in lower RMSE and log loss values than Parameters-1, then it means that this deck should have its own preset (Preset-2) with its own parameters (Parameters-2).
+
+Don't worry, FSRS will still perform well even without the aforementioned steps. This method is optional.
+
+Parameters are calculated from the review history of all decks the use the current preset. If you want to alter which cards are used for optimizing the parameters (such as excluding suspended cards), you can adjust the search before calculating the parameters. The search works the same way as it does in the Browser. For details, see [Searching](https://docs.ankiweb.net/searching.html) in the Anki Manual. Optimization doesn't happen automatically, you have to manually click "Optimize" or "Optimize all presets".
```
Comment (brishtibheja) [inline docs/tutorial.md:122]: Minor mistake. It shouldn't be "all decks the use" in the first sentence.

**Comment by Expertium** [comment]
> "This method also works for decks using the same preset but that do not share a parent-child relationship"

Then where do you get Parameters-1? From a completely unrelated preset?

**Comment by brishtibheja** [comment]
I have a deck called Organic Chemistry with one subdeck called Ketones. Both of them share the same preset. I can use this method to find out whether a seperate preset would benefit me. Now I seperate the decks. I don't put Ketones under the Organic Chemistry deck. They are two distinct decks with one preset. The parameters are optimised from the review history of both these decks.

> Then where do you get Parameters-1?

Why? Parameters will come from the shared Preset i.e., Preset-1. How you organise the decks in deck screen is unrelated.

**Comment by Expertium** [comment]
Can you write a step-by-step instructon? I think you would do a better job than me.

**Comment by brishtibheja** [comment]
Here is what I wrote:

 If you are confident in your knowledge of the algorithm and are willing to do extra work, there is a way to check whether a deck benefits from having it's own preset.

1. Go to `deck options` of the deck you want to create a new preset for. Now copy the parameters from the `FSRS parameters` field. We will use this later for evaluation.

2. Now create a new preset and save to this deck.

3. Paste the parameters you previously copied in the `FSRS parameters` field of the newly created preset.

4. Click `Evaluate` and write down the RMSE(bins) and log loss values.

5. Now click `Optimize` to obtain a new set of parameters. Click `Evaluate` and write down the RMSE(bins) and log loss values.

6. If this new set of parameters result in lower RMSE(bins) and log loss values then save the preset along with the new parameters.
---------------------------------------------------------------------------

my writing style is a bit different from yours. you can edit it as you see fit to maintain stylistic coherence. I also didn't use naming like preset-1/parameters-1.

**Comment by brishtibheja** [comment]
Also do you think people can get higher log loss but lower RMSE or vice versa?

**Comment by Expertium** [comment]
It's possible, yes.

**Review by L-M-Sherlock — APPROVED** [review]
Thanks for your contributions!

**Pull request merged**